src/Components/Results/Results.tsx

```diff
@@ -608,6 +608,14 @@ const Results = () => {
       );
     };
 
+    const currentCategory = categories.find((cat) => {
+      if (filt.category === false) {
+        return false;
+      }
+
+      return cat.defaultMessage === filt.category.value;
+    });
```

Comment:
The [find()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) array iterator method will either return the first element for which the statement is true or return `undefined`. What do you think about removing lines `612`-`614`?